spicy/runtime/src/tests/parser.cc

```diff
@@ -57,16 +57,18 @@ TEST_CASE("ParserPort") {
 
 TEST_CASE("atEod") {
     auto stream = hilti::rt::ValueReference<hilti::rt::Stream>();
+    auto filters = hilti::rt::StrongReference<filter::detail::Filters>();
 
     SUBCASE("empty") {
         bool expanding = false;
         SUBCASE("expanding view") { expanding = true; }
         SUBCASE("not expanding view") { expanding = false; }
 
-        CHECK_EQ(detail::atEod(stream, stream->view(expanding)), ! expanding);
+        // This would now yield to wait for input before deciding whether EOD has been reached.
+        // CHECK_EQ(detail::atEod(stream, stream->view(expanding), filters), ! expanding);
```

Comment:
Can we just remove this commented out part? "Now" will not be useful in the future.